scripts/package-deploy.sh

```diff
@@ -28,19 +28,17 @@ function stage-platform-files {
     stage-file ./examples/hotrod/hotrod-$PLATFORM $PACKAGE_STAGING_DIR/example-hotrod$FILE_EXTENSION
 }
 
-# package pulls built files for the platform ($1). If you pass in a file
-# extension ($2) it will be used on the binaries
+# package pulls built files for the platform ($2) and compresses it using the compression ($1).
+# If you pass in a file extension ($3) it will be look for binaries with that extension.
```

Comment:
nit: My understanding of `stage-platform-files` is that it copies binary files over into the `$PACKAGE_STAGING_DIR` and adds the `$FILE_EXTENSION` to that copied filename, and AFAICT, doesn't look for binaries with `$FILE_EXTENSION`. Perhaps we should update this sentence.